ui/css/perf.css

```diff
@@ -49,6 +49,11 @@ to { opacity: 0; }
   color: #999
 }
 
+.graph-legend.seriesinactive {
+  border-color: #D3D3D3;
+  color: #D3D3D3;
```

Comment:
I was just reading a couple days ago in the Google Style Guide, which suggests that css hex colors should be lower case for readability. Whether everyone agrees with that or not I don't know, but providing the info here.